internal/services/network/network_packet_capture_resource.go

```diff
@@ -60,6 +60,17 @@ func resourceNetworkPacketCapture() *pluginsdk.Resource {
 				ForceNew: true,
```

Comment:
We should add validation here to limit this to a vm or a vmss

Reply:
Updated